package.json

```diff
@@ -18,19 +18,18 @@
   },
   "devDependencies": {
     "chai": "^2.1.1",
-    "coffee-script": "1.6.3",
     "grunt": "^0.4.5",
     "grunt-cli": "^0.1.13",
     "grunt-contrib-watch": "~0.6.1",
     "grunt-mocha-test": "~0.12.7",
     "grunt-release": "~0.11.0",
-    "hubot": "2.x",
+    "hubot": "3.x",
```

Comment:
This should likely be removed in lieu of specifying a `peerDependencies` ([reference](https://nodejs.org/en/blog/npm/peer-dependencies/)).